lib/msal-core/tests/Authority.spec.ts

```diff
@@ -151,15 +153,15 @@ describe("Authority", () => {
         it("is thrown when tenant discovery endpoint fails with invalid data", (done) => {
             verifyError(done, {
                 status: 500,
-                responseText: 'fatalError'
-            }, "fatalError");
+                responseText: 'invalid_instance'
+            }, "invalid_instance");
         });
 
         it("is thrown when tenant discovery endpoint fails with error details", (done) => {
             verifyError(done, {
                 status: 400,
-                responseText: '{"error": "OMG_EPIC_FAIL"}'
-            }, "OMG_EPIC_FAIL");
+                responseText: '{"error": "invalid_instance"}'
```

Comment:
@manoj-rath @sameerag  I was talking about this error message. Earlier it was OMG_EPIC_FAIL and I am getting invalid_instance. I couldn't find any reference of OMG_EPIC_FAIL on internet. I have changed the test case for now.